src/seeds/class.js

```diff
@@ -1 +1,114 @@
-// seeds here
+import mongoose from 'mongoose';
+
+export default [
+  {
+    _id: new mongoose.Types.ObjectId('64667748fc13ae7f027543cb'),
+    hour: '2:11',
+    day: '04/03/2023',
+    trainer: [
+      '646596b54fcb63fdd73b28a6',
+      '6465558202739f6df0878ceb',
+    ],
+    activity: '6465992cc7fee6b84bae2696',
+    slots: 9,
+  },
+  {
+    _id: new mongoose.Types.ObjectId('64667748fc13ae7f027543cc'),
+    hour: '18:42',
+    day: '23/02/2023',
+    trainer: [
+      '646596b54fcb63fdd73b28a6',
+      '6465558202739f6df0878ceb',
+    ],
+    activity: '64661bf03cfd3f20a4b7db44',
+    slots: 4,
+  },
+  {
+    _id: new mongoose.Types.ObjectId('64667748fc13ae7f027543cd'),
+    hour: '3:14',
```

Comment:
The time written as 3:14 is not a valid format for the schema model. It should be written as 03:14.
Please update the time format for the rest of the classes as well
```suggestion
    hour: '03:14',
```